dark/assets/css/styles.css

```diff
@@ -752,7 +758,7 @@ html {
   margin: 50px;
   margin-bottom: 400px;
   display: flex;
-  margin-left: 460px;
+

```

Comment:
Can you delete this extra line @priyankarpal

Reply:
ok

Reply:
Can you delete this extra line @priyankarpal 
```suggestion
```

Reply:
````suggestion
````
Remove this line @priyankarpal

Reply:
Do these above mentioned changes @priyankarpal